src/Components/App/App.js

```diff
@@ -27,6 +27,7 @@ class App extends Component {
       <main className="App">
         <Switch>
           <Route exact path='/' render={() => <Login userInfo={this.userInfo}/> } />
+          <Route path='/movies' render={() => <Login userInfo={this.userInfo}/> } />
```

Comment:
I think here the component should be MoviesContainer rather than login

Reply:
@Asilo5 I fixed the path for the route. But commented out until the information in the MoviesContainer component it correct and the fetch is working.